messaging.rst

```diff
@@ -85,27 +85,80 @@ Messages
 Locked Transfer
 -----------------
 
-Cancellable and expirable :term:`transfer`. Sent by a node when a transfer is being initiated, this message adds a new lock to the list of pending locks from the sending participant node.
+A LockedTransfer is a message used to reserve tokens for a new mediated transfer.
 
-Invariants
-^^^^^^^^^^
+Preconditions
+^^^^^^^^^^^^^
 
-Only valid if all the following hold:
+For a ``LockedTransfer`` to be considered valid the following conditions have to be true.
 
 - There is a channel which matches the given :term:`chain id`, :term:`token network` address, and :term:`channel identifier`.
 - The corresponding channel is in the open state.
 - The :term:`nonce` is increased by ``1`` in respect to the previous :term:`balance proof`
 - The :term:`locksroot` must change, the new value must be equal to the root of a new tree, which has all the previous locks plus the lock provided in the message.
-- The :term:`locked amount` must increase, the new value is equal to the old value plus the lock's amount.
-- The lock's amount must be smaller then the participant's :term:`capacity`.
+- Have the new lock represented in merkleroot.
+- The :term:`locked amount` must increase by exactly ``lock.amount`` otherwise, the message would be rejected by the recipient. If the ``locked_amount`` is increased by more, then funds may get locked in the channel. If the ``locked_amount`` is increased by less, then the recipient will reject the message as it may mean it received the funds with an on-chain unlock. The initiator will stipulate the fees based on the available routes and incorporate it in the lock's amount. Note that with permissive routing it is not possible to predetermine the exact `fee` amount, as the initiator does not know which nodes are available, thus an estimated value is used..
```

Comment:
To me only the first sentence seems to be a precondition. The rest seems to be an explanation of the precondition and our fee model. Maybe we can find another place for that?

Reply:
If you think it's better. The text here was either in the original spec or copied from the python class docstrings.

Reply:
Do you think it is better? I think it would be more consistent, to list the preconditions and either explain them all or explain the process / reasons for it in another place. If you like to do this together, you could seperate those in the bullet itself. 
- (PC5) The :term:`locked amount` must increase by exactly ``lock.amount`` otherwise, the message would be rejected by the recipient. - Rationale: _If the ``locked_amount`` is increased by more, then funds may get locked in the channel. If the ``locked_amount`` is increased by less, then the recipient will reject the message as it may mean it received the funds with an on-chain unlock._

Reply:
I think you can have footnotes and terms in rst so we could make the description of the preconditions a simple summary, as simple as possible as long as it makes sense and then we can have a reference to an explanation. Do you think such an approach would make sense?